svn/common.py

```diff
@@ -214,15 +214,21 @@ def log_default(self, timestamp_from_dt=None, timestamp_to_dt=None,
         if stop_on_copy is True:
             args += ['--stop-on-copy']
 
+        if changelist is True:
+            args += ['--verbose']
+
         result = self.run_command(
                     'log', 
                     args + ['--xml', full_url_or_path], 
                     combine=True)
 
         root = xml.etree.ElementTree.fromstring(result)
+        named_fields = ['date', 'msg', 'revision', 'author']
+        if changelist is True:
+            named_fields += ['changelist']
```

Comment:
Put the "changelist" item in there no matter what. Just set it to None if not requested. We want an immutable type to always have the same shape.